source/includes/baremetal/devices.md

```diff
@@ -0,0 +1,690 @@
+      "ipmiAddress": "10.6.91.160",
+      "powerStatus": "ON",
+      "tags": [],
+      "hostname": "tester02.coxedge.com",
+      "location": {
+        "facility": "DAL1",
+        "facility_title": "Dallas 1"
+      }
+    },
+    {
+      "id": "13547",
+      "servicePlan": "149588",
+      "name": "testcc01.coxedge.com",
+      "deviceType": "Dedicated Server 1U",
+      "primaryIp": "66.165.229.26",
+      "status": "active",
+      "monitorsTotal": 0,
+      "monitorsUp": 0,
+      "ipmiAddress": "10.4.66.21",
+      "powerStatus": "ON",
+      "tags": ["ultra-wp"],
+      "hostname": "testcc01.coxedge.com",
+      "location": {
+        "facility": "LAX2",
+        "facility_title": "Los Angeles 2"
+      }
+    }
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/devices</code>
+
+Retrieve a list of all devices in a given [environment](#administration-environments).
+
+| Attributes                             | &nbsp;                                                                                       |
+| -------------------------------------- | -------------------------------------------------------------------------------------------- |
+| `id`<br/>_string_                      | The unique ID of the device.                                                                 |
+| `servicePlan`<br/>_string_             | The unique ID of the service associated with this device.                                    |
+| `name`<br/>_string_                    | User given custom name.                                                                      |
+| `deviceType` <br/>_string_             | Generic description of device. Usually type and rack unit size.                              |
+| `primaryIp` <br/>_string_              | The first assigned public IP for accessing this device.                                      |
+| `status` <br/>_string_                 | active/inactive                                                                              |
+| `monitorsTotal`<br>_int_               | Total # device monitors                                                                      |
+| `monitorsUp`<br/>_int_                 | Of passing device monitors.                                                                  |
+| `ipmiAddress` <br/>_string_            | IP address for IPMI connection. Requires you to whitelist your current IP or be on IPMI VPN. |
+| `powerStatus` <br/>_string_            | ON/OFF                                                                                       |
+| `tags`<br/>_Array[string]_             | List of all user set device tags.                                                            |
+| `hostname` <br/>_string_               | A FQDN for the device. for example: example.hivelocity.net.                                  |
```

Comment:
Should be "For example" with a capital f

Reply:
Resolved